accounts/fixtures/profiles.json

```diff
@@ -0,0 +1,2 @@
+Postgres DATABASE_URL not found, using db.sqlite3
```

Comment:
I think this file is not needed. I already added a file to `accounts/fixtures` to do this for one admin user and we probably don't need more.

Reply:
Sorry ignore this please. Just saw that you added quite a lot, so let's leave these in! :+1: